prime-router/src/main/kotlin/ValueSet.kt

```diff
@@ -40,6 +40,10 @@ data class ValueSet(
         val code: String,
         val display: String? = null,
         val version: String? = null,
+        // replaces is used in the case of an altValue that needs to be used instead
+        // of what is normally used in the valueSet.
+        // for example, a DOH might want to use 'U' instead of 'UNK' for Y/N/UNK values
+        val replaces: String? = null
     )
```

Comment:
AltValue is kind of a replaces of the main value set. When do we need a replaces vs an altValue? Isn't this a map?

Reply:
It would probably be easier to have this discussion in a meeting, but the idea here is for something like the Florida file where FL expects `Y, N, or U` instead of the `Y, N, UNK`. By specifying that `U` is a drop in for the `UNK` we can correctly translate values when they come in, either a `U` or a `UNK`

Reply:
So, just to document our conversation. We agree to accept `replace` for now, but to generalize altValues to work for mapping display and code values in the future.